gui/elements/qt/_imageLayer.py

```diff
@@ -5,13 +5,13 @@ class QtImageLayer(QtLayer):
     def __init__(self, layer):
         super().__init__(layer)
 
-        comboBox = self.children()[3]
-        for cmap in self.layer.colormaps:
-            comboBox.addItem(cmap)
-        index = comboBox.findText('hot', Qt.MatchFixedString)
-        if index >= 0:
-            comboBox.setCurrentIndex(index)
-        comboBox.activated[str].connect(lambda text=comboBox: self.changeColor(text))
+        #comboBox = self.children()[3]
```

Comment:
Why did this get disabled anyways?

Reply:
I removed the cmap selection from the main layer widget. It will now be accessible by double clicking on the layer widget along with other custom layer specific properties. I havn't added that back in yet, but it will be my next PR after this one is merged.

Reply:
Will double-clicking be replaced by a sandwich menu at some point?

Reply:
I'm open to either, I have created a new issue to discuss this #59. I think it will be in a new PR regardless, currently neither are implemented. It's one of the next things on my list though after this one get's merged.